packages/instant/src/components/buy_order_progress.tsx

```diff
@@ -21,15 +22,25 @@ export const BuyOrderProgress: React.StatelessComponent<BuyOrderProgressProps> =
         const progress = buyOrderState.progress;
         const hasEnded = buyOrderState.processState !== OrderProcessState.Processing;
         const expectedTimeMs = progress.expectedEndTimeUnix - progress.startTimeUnix;
-        return (
-            <Container padding="20px 20px 0px 20px" width="100%">
+        content = (
+            <React.Fragment>
                 <Container marginBottom="5px">
                     <TimeCounter estimatedTimeMs={expectedTimeMs} hasEnded={hasEnded} key={progress.startTimeUnix} />
                 </Container>
                 <TimedProgressBar expectedTimeMs={expectedTimeMs} hasEnded={hasEnded} key={progress.startTimeUnix} />
+            </React.Fragment>
+        );
+    } else {
+        // Just show a static progress bar if we aren't processing or in an end state
+        content = (
```

Comment:
Per slack convo, this feels a bit unnecessary, esp. since we never actually hit this scenario